### ISSUE TITLE
Setting date at the right with multicalendar

### ISSUE THREAD

**Issue by MLS535** [Vuepic/vue-datepicker#441]
This is more a question than a bug...
I created a datepicker with the following issues:

<Datepicker multi-calendars multi-calendars-solo class="flex-1" format="dd/MM/yyyy" v-model="rangeDate"
        :enable-time-picker="false" range auto-apply :partial-range="true"
         placeholder="add a range of dates" text-input
        :preset-ranges="presetRanges" @update:model-value="getOverviewhse()"
        :min-date="new Date(new Date().setFullYear(new Date().getFullYear() - 5))" :max-date="new Date()"
        prevent-min-max-navigation>
        <template #yearly="{ label, range, presetDateRange }">
          <span @click="presetDateRange(range)">{{ label }}</span>
        </template>
      </Datepicker> 
The problem is that I want today's date to always be on the right and the previous month on the left to avoid what appears in the [image ](https://i.stack.imgur.com/iTkZ9.png) that the right calendar is disabled. How can I do it?

In other words, for example, in the month of May, it should be on the left, and the month of June should be on the right...

**Comment by Jasenkoo** [comment]
You can achieve this with `start-date` and `focus-start-date` props. Take a look at this [example](https://stackblitz.com/edit/vuepic-vue-datepicker-qqjuwp?file=src%2Fcomponents%2FPlayground.vue).
